config/routes.rb

```diff
@@ -5,4 +5,5 @@
 
   root to: 'home#index'
   resources :picks, only: %i[index new create]
+  resources :standings, only: %i[index]
```

Comment:
We'll want to do this differently in the future (to allow viewing standings for past years and sports), but this is fine for now.

E.g. in the future we might want something like `/standings/nba/2021` rather than just `/standings`